tests/unit/tasks/server/express-server-test.js

```diff
@@ -284,8 +284,43 @@ describe('express-server', function() {
               });
           });
       });
-    }),
 
+      it('does not use compression for server sent events', function(done) {
+        project.require = function() {
+          let app = express();
+          app.use('/foo', function(req, res) {
+            res.set('Content-Type', 'text/event-stream');
+            res.send(longText);
+          });
+          return app;
+        };
+
+        subject.start({
+          proxy: 'http://localhost:3001/',
+          host: undefined,
+          port: '1337',
+          rootURL: '/',
+          compression: true,
+        })
+          .then(function() {
```

Comment:
Can we use co instead of the promise chain? and possibly promisify request?

Reply:
Good idea, done.